cartridges/rvw_dev_console/cartridge/controllers/Console.js

```diff
@@ -35,7 +35,8 @@ server.post(
     server.middleware.https,
     function (req, res, next) {
         if (dw.system.System.instanceType === dw.system.System.PRODUCTION_SYSTEM) {
-            res.setStatusCode(403).json({
```

Comment:
@z1haze I tested this during that login testing, and it looks like chaining these together does not correctly set the status code to 403.  Actual status code that was being returned was 410.  Had this issue in some code for another client so remembered this fixes it for this issue.  I think this is more related to headers not correctly being updated in chaining methods like this.